package.json

```diff
@@ -21,7 +21,6 @@
     "react-dom": "^16.13.1",
     "react-icons": "^3.9.0",
     "react-notification-alert": "0.0.12",
-    "react-query": "^1.3.3",
```

Comment:
`react-query` is managing a cache and state for our asynchronous data, so we don't hit the API repeatedly, our update loading and error states automatically. Would it be worth it to implement these changes keeping `react-query`?